src/popup/App.vue

```diff
@@ -132,6 +135,7 @@ export default defineComponent({
     const { loadCoinsData } = useCurrencies({ withoutPolling: true });
     const { restoreLanguage } = useLanguages();
     const { restore: restoreTransferSendForm } = useTransferSendHandler();
+    const { multisigAccounts } = useMultisigAccounts({ store });
```

Comment:
Running this composable here will force the mutisig accounts to be fetched everywhere in the app. The goal of `createPollingBasedOnMountedComponents` was to poll data only when it's needed. So for example the multisg accounts are not required in regular account details, all settings pages, signing modals etc...
So maybe you can add `disablePolling` setting to the composable?